arch/platform/cooja/dev/moteid.c

```diff
@@ -43,9 +45,13 @@ int simRandomSeed;
 static void
 doInterfaceActionsBeforeTick(void)
 {
+  struct csprng_seed csprng_seed;
```

Comment:
Contiki-NG is compiled as C99 nowadays, so you can avoid the memset in line 53 by doing `struct csprng_seed csprng_seed = {0};`. Also push the variable declaration inside the if-statement since it isn't used anywhere else in this function.